maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java

```diff
@@ -70,18 +76,26 @@ public class IncrementalifyMojo extends AbstractVersionsUpdaterMojo {
     @Component
     private BuildPluginManager pluginManager;
 
+    @Inject public IncrementalifyMojo(RepositorySystem repositorySystem, org.eclipse.aether.RepositorySystem aetherRepositorySystem, Map<String, Wagon> wagonMap, Map<String, ChangeRecorder> changeRecorders) {
+        super(repositorySystem, aetherRepositorySystem, wagonMap, changeRecorders);
+    }
```

Comment:
Adapting to the new recommended method for injecting components via constructor.